tm2py/components/network/transit/transit_assign.py

```diff
@@ -466,28 +543,167 @@ def transit_emmebank(self):
     def run(self):
         """Run transit assignments."""
 
-        use_ccr = False
-        if self.controller.iteration >= 0:  # TODO add option of warmstart
-            use_ccr = self.config.use_ccr
-
-            self.sub_components["prepare transit demand"].run()
-        else:
-            self.transit_emmebank.zero_matrix
         for time_period in self.time_period_names:
-            self.run_transit_assign(time_period, use_ccr)
+            if self.controller.iteration == 0:
+                use_ccr = False
+                congested_transit_assignment =  False
+                # update auto times
+                print("update_auto_times")
+                self.transit_network.update_auto_times(time_period)
+
+                # run extended transit assignment and skimming
+                # if run warm start, trim the demands based on extended transit assignment and run congested assignment
+                # otherwise run with 0 demands
+                if self.controller.config.run.warmstart.warmstart:
```

Comment:
@arashasadabadi: delete old code and confirm skims are being update from the warm start assignment.